packages/backend-app-api/src/wiring/BackendInitializer.ts

```diff
@@ -149,8 +150,31 @@ function createInstanceMetadataServiceFactory(
     .flat();
   return createServiceFactory({
     service: instanceMetadataServiceRef,
-    deps: {},
-    factory: async () => ({ getInstalledFeatures: () => installedFeatures }),
+    deps: {
+      httpRouter: coreServices.rootHttpRouter,
+      logger: coreServices.rootLogger,
+    },
+    factory: async ({ logger, httpRouter }) => {
+      const instanceMetadata = {
+        getInstalledFeatures: () => installedFeatures,
+      };
+
+      logger.info(
+        `Installed features on this instance: ${instanceMetadata
+          .getInstalledFeatures()
+          .join(', ')}`,
+      );
+
+      const router = Router();
+
+      router.get('/features/installed', (_, res) => {
+        res.json({ items: instanceMetadata.getInstalledFeatures() });
+      });
```

Comment:
Random though: I think it might be useful to also include the timestamp of when the instance was started, to help resolve conflicts in the metadata. That in turn made me think that maybe this is best designed as a single "fat" one-stop-shop endpoint where we put anything one might want to know. Just `/info` or something.